src/core/dfn.js

```diff
@@ -12,10 +12,9 @@ export function run() {
     const titles = getDfnTitles(dfn);
     registerDefinition(dfn, titles);
 
-    // Treat Internal Slots as IDL.
-    if (!dfn.dataset.dfnType && /^\[\[\w+\]\]$/.test(titles[0])) {
-      dfn.dataset.dfnType = "idl";
-    }
+    const [firstTitle] = titles;
```

Comment:
Should we call it `linkingText` instead?

Reply:
yeah, firstTitle is 🤢... why these are called "title" at all is beyond me.